src/Symfony/Component/BrowserKit/AbstractBrowser.php

```diff
@@ -138,6 +138,14 @@ public function getServerParameter(string $key, mixed $default = ''): mixed
         return $this->server[$key] ?? $default;
     }
 
+    /**
+     * Gets server parameters.
+     */
+    public function getServerParameters(): array
```

Comment:
Is this method only needed for the test?

Reply:
Yes